src/Sylius/Bundle/CoreBundle/Behat/MetadataContext.php

```diff
@@ -342,7 +340,7 @@ private function getFormWithFields(ElementInterface $element, array $fields)
             }
         }
 
-        return null;
+        return;
```

Comment:
of course. this is part of the symfony style. 
if you don't want this we can remove the rule.
Discuss and let me know,